docs/NEW_GUIDE_DRAFT.en.md

```diff
@@ -0,0 +1,57 @@
+login
+create-account
+    - implicit
+    - subaccount
+manage-account
+    <!-- Should we add local key management? -->
+    - keys
+        - view
+        - add
+        - delete
+        - nonce
+    - contract
+        - deploy
+        - dev-deploy
+        - state
+        - code
+        <!-- TODO: what is the best name for it? -->
+        - change-function-call
+        <!-- TODO: what is the best name for it? -->
+        - view-function-call
+    - state
+    - delete
+transfer
+<!-- TODO: should it be manage-account? Should it be here at all? If yes, where is `unstake`? Maybe it should be an extension, like "near-cli-extension-staking".  -->
```

Comment:
feature creep -- but what if we have a feature where we can have shortcuts? So to be consistent, these would live in `manage-account`, but we also have shortcuts to them?